framework/doc/content/templates/sqa/rtm.md.template

```diff
@@ -91,6 +91,24 @@ includes:
 - The steps to validate the test result including any acceptance criteria.
 !template-end!
 
+### Changelog Issue Revisions id=log-revisions
+
+Errors in changelog references can sometimes occur as a result of typos or conversion errors. If
+any need to be noted by the development team, they will be noted here.
+
+!template! field key=log-revisions
+If errors are found in issue references related to the changelog (e.g., commit history), these
+should be noted here. The commit in which the errant issue reference occurs should be noted, and the
+correct issue should be listed alongside each entry. Relevant links to all items are required.
+An example of this might include:
+
+!table
+| MOOSE Commit | Incorrect Referenced Issue | Correct Referenced Issue |
+| - | - | - |
+| [48db61](https://github.com/idaholab/moose/commit/48db61307ed87b58a96e944215f13378138cf7bc) | idaholab/sockeye#19845 | [idaholab/moose#19845](https://github.com/idaholab/moose/pull/19845) |
```

Comment:
note the second "correct" is not correct here. the correct sockeye issue has not been ported yet

Reply:
I guess comment mostly applis to the thm file not that one which is a template

Reply:
Do we have any commits that present a good initial entry for this in the module? I'll just use the same one for the example.

Reply:
idaholab/sockeye#130

Reply:
(is the entry for the "correct" column)

Reply:
@joshuahansel thanks for suggesting the correct one. I would also like to eventually have a connection to a ported MOOSE issue for idaholab/sockeye#130, as the latter might not be accessible by an auditor. I am OK with putting in a link to sockeye for the moment for the sake of example, while we get the discussion in the original issue sorted out and porting is completed.

Reply:
Ok, so does this mean we should just avoid referencing issues from other repos, or at least repos that have some kind of access restriction?

Reply:
It's OK to reference other issues in discussion for completeness, but we should avoid having non-MOOSE issues as the documented *sole* reason for a change. If, for example, a BISON issue/discussion spawned or served to influence a MOOSE issue, that's fine to link in as extra context, but we shouldn't be relying on that BISON issue as the only context or only issue.

Reply:
@joshuahansel I actually might have stumbled across a good example of an incorrect issue reference from the THM merge that I'd like to use instead here. See [f025970](https://github.com/idaholab/moose/commit/f0259701b3e961129678574d75d25a9c4fdec157), where I think `idaholab/thm#504` should be the ported issue `idaholab/moose#19694`. This isn't obvious at first glance, since this was ported from `idaholab/thm#16`, but it mentions `idaholab/relap-7#504` in the body of the issue. So I think David had a typo in the commit history even when it was merged into the THM repo?

Reply:
I think David meant to do `Ref idaholab/relap-7#504`. I see that in the preceding commit. `idaholab/moose#19694` appears to be a kind of sub-issue added later that doesn't quite fit the commit; `idaholab/relap-7#504` is probably most appropriate. However, it can probably be argued either way.

Reply:
Right, it does seem like he meant `relap-7` there as well. 19694 seems like it was a catch-all in the THM repo for 504 in RELAP-7, to me. 

Would you like the see the example take into account these sorts of typos as well? My goal was to mostly make sure that mangled or possibly-incorrect references were connected to relevant _MOOSE_ issues (which is most important from an SQA perspective), neglecting a bit the connection to the repo it might have come from (since that can be linked from a MOOSE issue).

Reply:
I guess it depends on whether we're going to go through the MOOSE repo and list out MOOSE issues anytime we find an application repo issue number instead of a MOOSE one. That might be a big task.

I would suggest that we start by just *correcting* issue numbers, even if still to an external repo, so that there's some hope of an auditor following the trail, even if that means they'd need access. Then in a later step, we can reference or create MOOSE issues instead. Are you sure that they won't have access to the others?

Reply:
Realistically - if they asked, they'd likely be guided through the answer by a developer who would have access. So it's probably OK to start here and adjust if necessary. I am likely overthinking it.

I'll remove the last commit and fix it up to the original correction for Sockeye, and I think we can run with this for now.